eclipse_formatter.xml

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
It is very hard to maintain source code if people dont agree on formatting. So yes, I think it is project related. It is the formatting we should enforce on any pull requests.